docs/products/kafka/howto/set-kafka-parameters.rst

```diff
@@ -0,0 +1,44 @@
+Manage Apache Kafka parameters
+===================================
+
+Aiven for Apache Kafka comes by default with a preset of configuration parameters defining for example automatic topic creation, partition count, replication factor, and retention time. These service configurations can be retrieved and managed via the Aiven web console, under the *Advanced configuration* section of the *Overview*
+page for your service, or using the ``-c`` options via the :ref:`Aiven CLI service update command <avn-cli-service-update>`.
+
+.. Warning::
+
+    Most of the Apache Kafka settings cause the service to restart when changed. Aiven for Apache Kafka restarts nodes one at a time to ensure minimal disruption to service availability. However, it can take few minutes from the change before the new settings are in use.
+
+Retrieve the current service parameters with Aiven CLI
+-----------------------------------------------------------
+
+To retrieve the existing Aiven for Apache Kafka configuration use the following command:
+
+::
+
+    avn service get SERVICE_NAME --json
+
+The output is the JSON representation of the service configuration.
+
+Retrieve the customizable parameters with Aiven CLI
+----------------------------------------------------------------
+
+Not all Aiven for Apache Kafka parameters are customizable, to retrieve the list use the following command:
```

Comment:
"to retrieve the list " - is it "to retrieve the full list" or "to retrieve the list of those parameters you can change"? I think better to specify what list we're talking about